lib/utils/error.js

```diff
@@ -42,6 +42,7 @@ function handleError(err, {debug = false} = {}) {
       err.forEach(err => error(err.message))
     } else {
       error(err)
+      process.exit(1)
```

Comment:
@Mikanebu can you please remove it? The `process.exit(1)` happens after a report is sent to Sentry. You're probably experiencing a buggy behaviour because you've setup `datahub=dev` which doesn't send reports so never hits `process.exit(1)` here - https://github.com/datahq/data-cli/blob/master/lib/utils/error.js#L25

Reply:
or does it break tests?

Reply:
@anuveyatsu Spinner keeps loading 
```
> Error! 404: Not Found. Requested URL: https://datahub.io/core/co2-ppm/r/15.csv
test1
> Error! Cannot read property 'pipe' of undefined
⠋ Loading...
```